xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1022,6 +1021,13 @@ void CXBMCApp::onReceive(CJNIIntent intent)
     bool newstate = m_headsetPlugged;
     if (action == "android.intent.action.HEADSET_PLUG")
       newstate = (intent.getIntExtra("state", 0) != 0);
+
+      // If we have unplugged headset and playing content with audio then try to pause
+      if (!newstate && (m_playback_state & PLAYBACK_STATE_PLAYING) && (m_playback_state & PLAYBACK_STATE_AUDIO))
+      {
+        CApplicationMessenger::GetInstance().PostMsg(TMSG_GUI_ACTION, WINDOW_INVALID, -1,	
+                                                      static_cast<void*>(new CAction(ACTION_PAUSE)));
+      }
```

Comment:
> Use spaces as tab policy with an indentation size of 2.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation

Reply:
I'm confused, all the indents above are 2 spaces?

Or are you talking about the linebreak on the static_cast?
That was just copied from the original merged PR
I see elsewhere in code there is no line break for the same line.
But that leads to length over 100.

Reply:
If you look at the build artifacts on jenkins at the top of the job, it always applies the clang-format as part of its build. So the diffs available will be of the clang-format changes. one for each commit, and then a whole PR single diff.

https://jenkins.kodi.tv/job/BuildMulti-PR/16106/

I find that the easiest way to "fix" these cosmetics things

Reply:
Oh, I see the issue.
there needs to be brackets after   if (action == "android.intent.action.HEADSET_PLUG")
That used to be a single line so none were required.
so without those - my indentation is indeed in-correct